Compiler/preproc/preprocessor.cpp

```diff
@@ -190,6 +190,7 @@ namespace Preprocessor {
                     size_t endOfString = text.FindChar(text[i],i);
                     if (endOfString == NOT_FOUND) //size_t is unsigned but it's alright
                     {
+                        LogError(ErrorCode::UserDefinedError, "Unterminated string");
```

Comment:
Same here, `UserDefinedError` is only for #error preprocessor commands, not for actual user's mistakes.